mbed_devices/_internal/resolve_target.py

```diff
@@ -1,19 +1,25 @@
 """Resolve targets for CandidateDevice.
 
-Filesystem access is slow - this module does its best to do it lazily.
+Resolving a target involves looking up an MbedTarget from the mbed-targets API, using data found in the "htm file"
```

Comment:
If I were coming to this for the first time I wouldn't necessarily know what 'mbed-targets API' meant. Would it make sense to have a link to the mbed-targets repository here to illustrate what you mean?

Reply:
Yeah, I think it would. I'll add a link.